code-challenges.js

```diff
@@ -1,41 +1,105 @@
 // ASSESSMENT 3: Coding Practical Questions
 
-
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
-
-
-
+console.log("1:");
+
+const fibnachoSequence = (n) => {
+    // initializing an array to hold the beginning values of the sequence and an anchor for our loop
+    let fibonacciArray = [1, 1], i = 2
+    // While loop that pushes back to the array and adds the numbers according to the algorithm: X(n) = X(n-1) + X(n-2)
+    while(i < n){
+        fibonacciArray.push(fibonacciArray[i - 2] + fibonacciArray[i - 1]);
+        i++;
+    }
+    //return new array
+    return fibonacciArray
+}
```

Comment:
really cool solution! this one is a thoughy and you came up with something really cool.